Pcap++/src/PcapLiveDevice.cpp

```diff
@@ -261,6 +261,17 @@ pcap_t* PcapLiveDevice::doOpen(const DeviceConfiguration& config)
 	}
 #endif
 
+#ifdef HAS_DIRECTION_SET
+    ret = pcap_setdirection(pcap, config.direction);
+	if (ret == 0)
+	{
+	    LOG_DEBUG("Direction is set");

```

Comment:
maybe it's worth logging the direction itself also (IN, OUT, INOUT)

Reply:
Ok. I'll do it.